crawl-ref/source/exclude.cc

```diff
@@ -128,9 +128,20 @@ void remove_auto_exclude(const monster* mon, bool sleepy)
 travel_exclude::travel_exclude(const coord_def &p, int r,
                                bool autoexcl, string dsc, bool vaultexcl)
     : pos(p), radius(r),
-      los(los_def(p, opc_excl, circle_def(r, C_SQUARE))),
       uptodate(false), autoex(autoexcl), desc(dsc), vault(vaultexcl)
 {
+    //If the monster can't move, and doesn't do anything
+    //to you behind transparent walls, don't exclude beyond 
+    //them.
+    if (dsc == "oklob plant"
```

Comment:
The coord_def is available, it would be better check if there is a monster at the grid and check mons.is_stationary() for that monster in this conditional instead of these special cases.

Reply:
Requested changes made.